analyzers/src/SonarAnalyzer.ShimLayer/Syntax.xml

```diff
@@ -1987,9 +1987,7 @@
             <Kind Name="InterpolatedSingleLineRawStringStartToken"/>
             <Kind Name="InterpolatedMultiLineRawStringStartToken"/>
             <PropertyComment>
-                <summary>
-                    The first part of an interpolated string, <c>$"</c> or <c>$@"</c> or <c>$"""</c>
-                </summary>
+                <summary>The first part of an interpolated string, <c>$"</c> or <c>$@"</c> or <c>$"""</c></summary>
```

Comment:
This change and the next one
```cs
<summary>
                    This could be a single <c>{</c> or multiple in a row (in the case of an interpolation in a raw interpolated string).
                </summary>
                <summary>This could be a single <c>{</c> or multiple in a row (in the case of an interpolation in a raw interpolated string).</summary>
```
seems to not be part of the XML [release/dev17.12](https://github.com/dotnet/roslyn/blob/release/dev17.12/src/Compilers/CSharp/Portable/Syntax/Syntax.xml), can you please double check?